### ISSUE TITLE
Translations in Transifex

### ISSUE THREAD

**Issue by Valdnet** [nextcloud/nextcloud-vue#2062]
I noticed that there are a lot of text strings that are not translated in Transifex. The lack of translations results in a slightly underdeveloped NC instance.

@skjnldsv Is it possible to easily add these strings to be translated? This is a priority issue 😉.

**Comment by skjnldsv** [comment]
I think this is the issue where all translations fail to be extracted from the templates.
I think we could maybe consider extracting the strings from the dev builds instead of source :thinking:

**Comment by raimund-schluessler** [comment]
@Valdnet Is this still the case? If so, could you show the strings that are not translated, please?

**Comment by Valdnet** [comment]
@raimund-schluessler There are still many text strings that are not translated in Transifex. You must review all of your code to catch these text strings as they are in different files.

**Comment by raimund-schluessler** [comment]
> @raimund-schluessler There are still many text strings that are not translated in Transifex. You must review all of your code to catch these text strings as they are in different files.

So they are on Transifex already, just not translated? This is nothing we can solve here, in that case the translators on Transifex need to catch up with the strings.

**Comment by Valdnet** [comment]
> So they are on Transifex already, just not translated?

@raimund-schluessler No. The text strings are not adapted at all for the Transifex bot to scan them and add them to the translation.

**Comment by raimund-schluessler** [comment]
> @raimund-schluessler No. The text strings are not adapted at all for the Transifex bot to scan them and add them to the translation.

Could you then please name the strings for which you noticed it? Saying "somewhere in your code" is not very helpful 🙈

**Comment by raimund-schluessler** [comment]
@Valdnet Some strings I fixed (more or less recently), see https://github.com/nextcloud/nextcloud-vue/pull/2229/commits/05171d8acd3a2a602864386efb93a8977c32626e So I wonder what else is missing. Please note, these translation are on transifex, but might not be released and propagated to the apps yet.

**Comment by Valdnet** [comment]
> Some strings I fixed (more or less recently), see 05171d8 

👍

> Please note, these strings to translate are on transifex, but might not be released and propagated to the apps yet.

Okay, then let's wait for them to be added to Transifex, and then see what it will be like in the instance 😉.

**Comment by Valdnet** [comment]
Still no text strings in Transifex with issue: #512.

**Comment by susnux** [comment]
After #4272 following string are missing:

```
Open contact menu
Clear search
Actions for item with name "{name}"
Open link to "{resourceName}"
Write message, use "@" to mention someone, use ":" for emoji autocompletion …
Load more "{options}""
```

Got this by just a quick search on the source code:
```bash
while read p; do
    grep "\"$p\"" l10n/messages.pot 2>&1 > /dev/null || echo "$p";
done < <(grep -hRPo "(?<=\bt\(')[^']+" src)
```
